test/functional/tests/fault_injection/test_fault_injection_interrupts.py

```diff
@@ -532,3 +547,29 @@ def create_test_file():
     dd.run()
     test_file.refresh_item()
     return test_file
+
+
+def wait_for_flushing(cache: Cache, core: Core, timeout: timedelta = timedelta(seconds=30)):
+    start_time = datetime.now()
+    while start_time + timeout > datetime.now():
+        try:
+            casadm_parser.get_flushing_progress(cache.cache_id, core.core_id)
+            return
+        except CmdException:
+            continue
+    raise CmdException("Management flush not started!")
+
+
+def wait_for_valid_dirty_blocks(volume, prev_stats,
+                                timeout: timedelta = timedelta(seconds=10)):
+    """
+    Necessary only in tests with the immediate interrupt because on some machines statistics read
+    by Test Framework are out-of-date and waiting few seconds solves the problem.
```

Comment:
Do we know why is that?

Reply:
I have to admit I do not. I've set breakpoints and checked the statistics on DUT - there were changes, but in test I needed to wait up to 4 seconds. I ran test from PyCharm, but I'll try with command line, maybe IDE has some dummy cache.

Reply:
@robertbaldyga Please review again, it would be good to have this changes:)

Reply:
My comment on that is still the same: I don't like this change. This is ugly workaround for a problem that exists somewhere else, be it test-framework or CAS itself. We expect stats to be updated immediately, so if they are not, then we'd rather like to have a test that fails than a test that introduces weird quirks only to obscure this problem.

Reply:
I've removed this method - during my tests on different platforms and OSes the problem described has not reappeared.